packages/types/src/Moment.ts

```diff
@@ -40,7 +38,7 @@ export default class Moment extends Base<Date> {
     );
   }
 
-  get bitLength (): UIntBitLength {
+  bitLength (): UIntBitLength {
```

Comment:
Is there a reason why these are now not getters? Additionally, I believe in `U8aFixed` we still have it as a getter?

Reply:
I would love to have bitLength everywhere as getter too, but on BN, bitLength is a method, so on Moment it needs to be a method too.

I'll change bitLenght on U8aFixed to method too, for consistency.

Reply:
Alternatively, @ts-ignore in Uint, but we stray away from BN

Reply:
No, rather keep it as a method, it is cleaner.